### PR TITLE
Small fixes for MQTT Serializer and MQTT Keep-alive Demos

### PR DESCRIPTION
**Pull request by sarenameas**
- Delete the leading underscore in the milliseconds helper macros.
- Add articles to some comments.
- Add clarify to come comments.
- Delete the extraneous line endings of log statements.
- Delete unused headers in the keep-alive demo.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c [hunk 1]
```diff
@@ -468,44 +472,44 @@ static void prvMQTTDemoTask( void * pvParameters )
...
-            LogInfo( ( "Publish to the MQTT topic %s.\r\n", mqttexampleTOPIC ) );
+            LogInfo( ( "Publish to the MQTT topic %s.", mqttexampleTOPIC ) );
             prvMQTTPublishToTopic( &xMQTTContext );
 
-            /* Process incoming publish echo, since application subscribed to the
-             * same topic the broker will send publish message back to the
-             * application. */
-            LogInfo( ( "Attempt to receive publish message from broker.\r\n" ) );
+            /* Process incoming publish echo, since the application subscribed to
+             * the same topic the broker will send the same publish message back
```
Comment (yourslab) [inline FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c:488]:
```suggestion
             * the same topic, the broker will send the same publish message back
```

File: FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c [hunk 2]
```diff
@@ -139,9 +139,9 @@
 /**
  * @brief Keep alive time reported to the broker while establishing an MQTT connection.
  *
- * It is the responsibility of the Client to ensure that the interval between
- * Control Packets being sent does not exceed the this Keep Alive value. In the
- * absence of sending any other Control Packets, the Client MUST send a
+ * It is the responsibility of the client to ensure that the interval between
+ * control packets being sent does not exceed the this keep-alive value. In the
```
Comment (yourslab) [inline FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c:143]:
```suggestion
 * control packets being sent does not exceed this keep-alive value. In the
```

File: FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c [hunk 3]
```diff
@@ -400,7 +402,8 @@ static MQTTFixedBuffer_t xBuffer =
 /*-----------------------------------------------------------*/
 
 /**
- * @brief Create the task that demonstrates the Plain text MQTT API Demo.
+ * @brief Create the task that demonstrates the MQTT API over a plaintext TCP
```
Comment (yourslab) [inline FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c:405]:
```suggestion
 * @brief Create the task that demonstrates the coreMQTT API over a plaintext TCP
```

File: FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c [hunk 4]
```diff
@@ -468,44 +472,44 @@ static void prvMQTTDemoTask( void * pvParameters )
...
 
-        /* Process Incoming packet from the broker. */
+        /* Process an incoming packet from the broker. */
         xMQTTStatus = MQTT_ReceiveLoop( &xMQTTContext, mqttexampleRECEIVE_LOOP_TIMEOUT_MS );
         configASSERT( xMQTTStatus == MQTTSuccess );
 
         /**************************** Disconnect. *****************************/
 
-        /* Send an MQTT Disconnect packet over the already connected TCP socket.
+        /* Send an MQTT disconnect packet over the already connected TCP socket.
```
Comment (yourslab) [inline FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c:509]:
```suggestion
        /* Send an MQTT disconnect packet over the connected TCP socket.
```

File: FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c [hunk 5]
```diff
@@ -687,9 +692,9 @@ static void prvMQTTSubscribeWithBackoffRetries( MQTTContext_t * pxMQTTContext )
          * subscribe packet then waiting for a subscribe acknowledgment (SUBACK).
          * This client will then publish to the same topic it subscribed to, so it
          * will expect all the messages it sends to the broker to be sent back to it
-         * from the broker. This demo uses QOS0 in Subscribe, therefore, the Publish
+         * from the broker. This demo uses QOS0 in Subscribe, therefore, the publish
```
Comment (yourslab) [inline FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c:695]:
```suggestion
         * from the broker. This demo uses QOS0 in Subscribe. Therefore, the publish
```

File: FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c [hunk 6]
```diff
@@ -700,18 +705,19 @@ static void prvMQTTSubscribeWithBackoffRetries( MQTTContext_t * pxMQTTContext )
...
         configASSERT( xResult == MQTTSuccess );
 
-        /* Check if recent subscription request has been rejected. #xTopicFilterContext is updated
-         * in the event callback to reflect the status of the SUBACK sent by the broker. It represents
-         * either the QoS level granted by the server upon subscription, or acknowledgement of
-         * server rejection of the subscription request. */
+        /* Check if the recent subscription request has been rejected. #xTopicFilterContext 
+         * is updated in the event callback to reflect the status of the SUBACK
+         * sent by the broker. It represents either the QoS level granted by the
+         * server upon subscription, or acknowledgement of server rejection of the
```
Comment (yourslab) [inline FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c:719]:
```suggestion
         * server upon subscription or acknowledgement of server rejection of the
```

File: FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c [hunk 7]
```diff
@@ -766,7 +772,7 @@ static void prvMQTTUnsubscribeFromTopic( MQTTContext_t * pxMQTTContext )
     MQTTStatus_t xResult;
     MQTTSubscribeInfo_t xMQTTSubscription[ mqttexampleTOPIC_COUNT ];
 
-    /* Some fields not used by this demo so start with everything at 0. */
+    /* Some fields are not used by this demo so start with everything at 0. */
```
Comment (yourslab) [inline FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c:775]:
```suggestion
    /* Some fields are not used by this demo, so start with everything at 0. */
```

File: FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c (hunk 1 above)
Comment (sarenameas) [reply FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c:488]: Thanks

File: FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c (hunk 2 above)
Comment (sarenameas) [reply FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c:143]: Hmmm I feel like this is nice here above the macro.

**Review by yourslab — APPROVED** [review]

**Review by cobusve — APPROVED** [review]

**Pull request merged**